src/bindings/js/node/include/model_wrap.hpp

```diff
@@ -76,6 +76,23 @@ class ModelWrap : public Napi::ObjectWrap<ModelWrap> {
      */
     Napi::Value get_outputs(const Napi::CallbackInfo& info);
 
+    /**
+     * @brief Sets a friendly name for a model.
+     * @param info Contains information about the environment and passed arguments
+     * this method accepts only one argument of type string ,
+     * throws ov::Exception if more than 1 arguments are provided or the provided argument is not of type String
+     * @return Null
```

Comment:
It's better to return Napi::Undefined in case when it's analogue of void. Because `typeof null = 'object'` in terms of JS.

Reply:
I see, you return Undefined already. Need to change here only.

Reply:
```suggestion
     * @return Napi::Undefined
```